langchain4j-jlama/src/test/java/dev/langchain4j/model/jlama/JacksonJsonCodecTest.java

```diff
@@ -0,0 +1,102 @@
+package dev.langchain4j.model.jlama;
+
+import org.junit.jupiter.api.Test;
+
+import static org.assertj.core.api.Assertions.*;
+
+import java.time.LocalDate;
+import java.time.LocalDateTime;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.List;
+
+
+class JacksonJsonCodecTest {
+
+    @Test
+    public void test() {
```

Comment:
please make sure tests here are realistic (they should deserialize `Map<String, Object>` where value can be `String`, `Example`, `LocalDateTime`, etc)